lib/config/index.ts

```diff
@@ -31,6 +31,22 @@ export function getManagerConfig(
   return managerConfig;
 }
 
+export function removeGlobalConfig(
```

Comment:
you probably should add a test to the config folder, so it's better self covered

Reply:
Added